content/Education and Kits/Registration/Access-and-share-course-content.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Access and share course content"
+id: 360021587259
+---
+
+When signed into Arduino Cloud you can access the following kits:
+
+* Kits that are registered to your private space.
+* Kits that are shared with a **shared space** of which you are a member.
+
+---
+
+## Access your courses
+
+Follow these steps:
+
+1. Go to [cloud.arduino.cc/home](https://cloud.arduino.cc/home).
+
+2. Use the space selection menu to switch between your spaces.
```

Comment:
```suggestion
2. Use the space selection menu (located at the top) to switch between your spaces.
```
I suggest explaining where the menu is located at least once in the article since it is a bit difficult to find it if one is not used to the cloud naming conventions.